must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -135,6 +133,32 @@ for ns in $namespaces; do
              pids_ceph+=($!)
         done
 
+         # Collecting output for ceph subvolume output
+        printf "collecting command output for: ceph fs ls\n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+        # Inspecting cephfs volume name
+        vol_name=$(timeout 120 oc -n openshift-storage exec "${HOSTNAME}"-helper -- bash -c "ceph fs ls"|awk '{print $2}')
```

Comment:
This is fs name not vol name, can we change the variable name to fsName or something more apt?

Reply:
We can also use this logic https://github.com/red-hat-storage/ocs-operator/blob/439ec42da6e644958d8d27ac7061471c4d27d55d/must-gather/collection-scripts/gather_ceph_resources#L247